pkg/auth/token.go

```diff
@@ -89,7 +95,7 @@ func Token(
 		"email": claims.Email,
 		"name":  claims.Name,
 		"roles": claims.Roles,
-		"iat":   time.Now().Unix(),
+		"iat":   time.Now().Add(-options.IATSubtract).Unix(),
```

Comment:
What is the default value of time.Duration?   Is this going to be ok: https://stackoverflow.com/questions/23051973/what-is-the-zero-value-for-time-time-in-go?

Reply:
Time.Duration is just an int64 alias, so it should be 0. So it should have no effect if it's not.

Reply:
Ah ok..  cool..